asapdiscovery-ml/asapdiscovery/ml/utils.py

```diff
@@ -7,27 +7,33 @@
 
 
 def build_dataset(
-    in_files,
     model_type,
     exp_fn,
+    all_fns=[],
+    compounds=[],
     achiral=False,
     cache_fn=None,
     grouped=False,
     lig_name="LIG",
     num_workers=1,
     rank=False,
+    str_only=False,
```

Comment:
str_only is a bit confusing as I think `string`, perhaps `structure_only`?

Reply:
that's reasonable. done in [ad31008](https://github.com/choderalab/asapdiscovery/pull/198/commits/ad31008b71b96fbf8e4fd1c56093840d822800cf)